.github/cosign.key

```diff
@@ -0,0 +1,11 @@
+-----BEGIN ENCRYPTED COSIGN PRIVATE KEY-----
+eyJrZGYiOnsibmFtZSI6InNjcnlwdCIsInBhcmFtcyI6eyJOIjozMjc2OCwiciI6
+OCwicCI6MX0sInNhbHQiOiJIYm5Zeno2c1orRytYdlFTUWorTU5PZEhnTmZTQnpR
+NTd4MkRIQWI5emU4PSJ9LCJjaXBoZXIiOnsibmFtZSI6Im5hY2wvc2VjcmV0Ym94
+Iiwibm9uY2UiOiIyS1pvUHF1bG9NcDUvcFBsOWg5cDR5VXBsL2M5eFI1OCJ9LCJj
+aXBoZXJ0ZXh0IjoiT1NqZUMvS2dtWUkzQ2ErVlVmQlh1Wm9hU0FkYWxFT0wwWk9G
+UEMrNFFWYWhtMUtNeHM2YUUwNWpvT3hveEF1eDRxaGk2amJmenp0MG5SelhJUUZt
+QjRSblBDTUQ4NmduQ2owR243dE4vc3V0TmpZbVI0c3NORzZpNXVYdTBuWmdseHk3
+K1k5SXU0cW0wOWordXRyNURwODM3RmF2Z0w3ZUhJeU1LQjlZWVd0OWZMV0s4VFps
+b29yTjJpVDYxT1E4Y0diM0JyOGw2ang2YkE9PSJ9
+-----END ENCRYPTED COSIGN PRIVATE KEY-----
```

Comment:
I'm a little nervous about having this key here, yet I assume it's inevitable? :)

Reply:
This key is just a placeholder.. the production key can be either in the repository or in the github secrets. This is one of the things we should discuss. The cosign team has their private key in their repository.

Reply:
@SantiagoTorres I have modified the PR accordingly. Now we are embedding the private key via github secrets instead of having it in the github repository. This adds another layer of 'security' (as long the github servers are not being tampered with).